upcloud/testdata/upcloud_kubernetes/kubernetes_s2.tf

```diff
@@ -23,17 +23,18 @@ resource "upcloud_network" "main" {
 }
 
 resource "upcloud_kubernetes_cluster" "main" {
-  name    = var.name
-  network = upcloud_network.main.id
-  zone    = var.zone
+  control_plane_ip_filter = ["0.0.0.0/0"]
```

Comment:
Could also test changing this value similarly than node-group node count change is now tested.

Reply:
Thanks, added a check for empty set.